kiali/kiali
# Fixed validation message position, fixed failing KIA1102 validation.

### Describe the change

Fixed failing cypress test 'KIA1102 validation'
Also fixed the validation messages position in YAML editor. Which was introduced after ' - manager:' fields are added in YAML editor inside collapsed "managedFields:".
Before:
![Screenshot from 2024-07-09 11-04-32](https://github.com/kiali/kiali/assets/604313/6acef8be-4ef9-494b-ac1d-d926f95a89ef)

After:
![Screenshot from 2024-07-09 11-06-03](https://github.com/kiali/kiali/assets/604313/35f5ab7e-b8bc-48ee-bfb3-e43e32aa9284)

Added missing "cluster" param in ValidationKey and Validations.

### Steps to test the PR
Make sure you have a VS or any other objects with validation messages or warnings.
Check that all messages in YAML editor are shown in correct lines.

### Automation testing

fixed failing test

### Issue reference
https://github.com/kiali/kiali/issues/7522
